reference/storage-class-parameters.md

```diff
@@ -2,6 +2,12 @@
 
 Storage class resource in Kubernetes is used to supply parameters to volumes when they are created. It is a convenient way of grouping volumes with common characteristics. All parameters take a string value. Brief explanation of each supported Mayastor parameter follows.
 
+
+{% hint style="info" %}
+The storage class parameter `local` has been deprecated in Mayastor version 2.0.
```

Comment:
Sorry to have not raised it earlier during your edit cycle but I'm only just thinking that we should probably call this out explicitly as a breaking change, and make it clear that it should not be used.